tha2/app/manual_poser.py

```diff
@@ -321,7 +321,7 @@ def update_result_image_panel(self, event: wx.Event):
         pose = torch.tensor(current_pose, device=self.device)
         output_index = self.output_index_choice.GetSelection()
         output_image = self.poser.pose(self.torch_source_image, pose, output_index)[0].detach().cpu()
-        numpy_image = convert_output_image_from_torch_to_numpy(output_image)
+        numpy_image = numpy.uint8(numpy.rint(convert_output_image_from_torch_to_numpy(output_image) * 255.0))

```

Comment:
Now that convert_output_image_from_torch_to_numpy will output a uint8 tensor, this conversion is not needed. Please revert.